### DISCUSSION TITLE
Not able to create a network using @aws-sdk/client-managedblockchain

### DISCUSSION THREAD

**Discussion by ashwinimiri** [aws/aws-sdk-js-v3#4360] [Q&A]
i am using @aws-sdk/client-managedblockchain to create a hyperledger network, but i got "InvalidRequestException: Invalid request body" 
<code>
const params = {
   ClientRequestToken : null,
   Framework: 'HYPERLEDGER_FABRIC', /* required */
   FrameworkVersion: '2.2', /* required */
   MemberConfiguration: { /* required */
     FrameworkConfiguration: { /* required */
       Fabric: {
         AdminPassword: 'adminUserName', /* required */
         AdminUsername: 'Adminpassword' /* required */
       }
     },
     Name: 'dummy Network', /* required */
     Description: 'testing',
     KmsKeyArn : null,
     LogPublishingConfiguration: null,
     Tags : null
   },
   Name: 'myNetwork', /* required */
   Tags : null,
   VotingPolicy: { /* required */
     ApprovalThresholdPolicy: {
       ProposalDurationInHours: 5,
       ThresholdComparator: 'GREATER_THAN' ,
       ThresholdPercentage: 50
     }
   },
   Description: 'testing',
   FrameworkConfiguration: {
     Fabric: {
       Edition: 'STARTER'
     }
   }
};

const createChannel = async ()=>{
    try
    {
         
        const command = new CreateNetworkCommand(params);
        const res = await client.send(command)
        console.log("response", res);
    }
    catch(ex){
        console.log(ex);
    }
     finally {
         client.destroy();
    }
}

createChannel();
<code>

**Comment by mnj1222** [comment]
I'm having a similar issue. Did you ever get this straightened out?

**Reply by mnj1222** [reply]
nm. figured out my issue.
